scripts/v.db.dropcolumn/v.db.dropcolumn.py

```diff
@@ -37,12 +37,13 @@
 # % required: yes
 # %end
 
-import string
 import grass.script as grass
 from grass.exceptions import CalledModuleError
 
 
 def main():
+    from grass.script.db import DBHandler
```

Comment:
Any reason for lazy import? The current DBHandler does the tricky stuff in instantiation, not import time.